js/main.js

```diff
@@ -12,15 +15,97 @@ function handlePhotoInput(event) {
 function handleSubmit(event) {
   event.preventDefault();
   const inputObj = {};
-  for (let i = 0; i < Array.from(event.target).length - 1; i++) {
-    inputObj[event.target[i].id] = event.target[i].value;
-  }
+  inputObj.title = event.target[0].value;
+  inputObj.photo = event.target[1].value;
+  inputObj.notes = event.target[2].value;
   inputObj.entryID = data.nextEntryId++;
   data.entries.unshift(inputObj);
   $form.reset();
   $photo.setAttribute('src', 'images/placeholder-image-square.jpg');
   $photo.setAttribute('alt', 'placeholder image');
+  viewSwap('entries');
+  toggleNoEntries();
+  $entriesList.prepend(renderEntry(inputObj));
+}
+
+function renderEntry(entry) {
+  const $listItem = document.createElement('li');
+  const $row = document.createElement('div');
+  $listItem.appendChild($row);
+  $row.className = 'row';
+
+  const $imageColumn = document.createElement('div');
+  $imageColumn.className = 'column-half entries-image-wrapper';
+  $row.appendChild($imageColumn);
+
+  const $textColumn = document.createElement('div');
+  $textColumn.className = 'column-half';
+  $row.appendChild($textColumn);
+
+  const $image = document.createElement('img');
+  $image.setAttribute('src', entry.photo);
+  $imageColumn.appendChild($image);
+
+  const $titleHeader = document.createElement('h3');
+  $textColumn.appendChild($titleHeader);
+
+  const $titleHeaderText = document.createTextNode(entry.title);
+  $titleHeader.appendChild($titleHeaderText);
+
+  const $paragraph = document.createElement('p');
+  $textColumn.appendChild($paragraph);
+  const $paragraphText = document.createTextNode(entry.notes);
+  $paragraph.appendChild($paragraphText);
+
+  return $listItem;
+}
+
+function toggleNoEntries() {
+  const $noEntries = document.querySelector('.no-entries');
```

Comment:
`querySelector`s should always try to be done globally. By putting these inside of a function, you're forcing us to traverse the DOM over and over everytime this function gets called. By moving this globally, we only have to do the search once, and we will always have a reference to it